wordle-web/pages/game.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+    <v-row>
+        <v-col>
+            <v-card>
+
+            <h2>Game Same</h2>
+            <p>
+                this is a game that we play with words and ideas to find the right answer.
+               
+            </p>
```

Comment:
```suggestion

            <v-card-title class="text-h2">Game Same</v-card-title>
            <v-card-text>
                this is a game that we play with words and ideas to find the right answer.
            </v-card-text>
```

Be sure to take advantage of the Vuetify text and typography classes as well as the Vuetify components for text such as `v-card-text` or `v-card-title`